test/fixtures/metafields.json

```diff
@@ -19,6 +19,486 @@
+      "updated_at": "2017-04-24T20:14:24.032-04:00",
+      "namespace": "namespace4",
+      "id": "72544103040",
+      "value": "value40",
+      "description": "This is metafield key40:value40 for namespace4",
+      "key": "key40",
+      "value_type": "string"
+    },
+    {
+      "created_at": "2017-04-24T20:14:24.032-04:00",
+      "updated_at": "2017-04-24T20:14:24.032-04:00",
+      "namespace": "namespace4",
+      "id": "72544103041",
+      "value": "value41",
+      "description": "This is metafield key41:value41 for namespace4",
+      "key": "key41",
+      "value_type": "string"
+    },
+    {
+      "created_at": "2017-04-24T20:14:24.032-04:00",
+      "updated_at": "2017-04-24T20:14:24.032-04:00",
+      "namespace": "namespace4",
+      "id": "72544103042",
+      "value": "value42",
+      "description": "This is metafield key42:value42 for namespace4",
+      "key": "key42",
+      "value_type": "string"
+    },
+    {
+      "created_at": "2017-04-24T20:14:24.032-04:00",
+      "updated_at": "2017-04-24T20:14:24.032-04:00",
+      "namespace": "namespace4",
+      "id": "72544103043",
+      "value": "value43",
+      "description": "This is metafield key43:value43 for namespace4",
+      "key": "key43",
+      "value_type": "string"
+    },
+    {
+      "created_at": "2017-04-24T20:14:24.032-04:00",
+      "updated_at": "2017-04-24T20:14:24.032-04:00",
+      "namespace": "namespace4",
+      "id": "72544103044",
+      "value": "value44",
+      "description": "This is metafield key44:value44 for namespace4",
+      "key": "key44",
+      "value_type": "string"
+    },
+    {
+      "created_at": "2017-04-24T20:14:24.032-04:00",
+      "updated_at": "2017-04-24T20:14:24.032-04:00",
+      "namespace": "namespace4",
+      "id": "72544103045",
+      "value": "value45",
+      "description": "This is metafield key45:value45 for namespace4",
+      "key": "key45",
+      "value_type": "string"
+    },
+    {
+      "created_at": "2017-04-24T20:14:24.032-04:00",
+      "updated_at": "2017-04-24T20:14:24.032-04:00",
+      "namespace": "namespace4",
+      "id": "72544103046",
+      "value": "value46",
+      "description": "This is metafield key46:value46 for namespace4",
+      "key": "key46",
+      "value_type": "string"
+    },
+    {
+      "created_at": "2017-04-24T20:14:24.032-04:00",
+      "updated_at": "2017-04-24T20:14:24.032-04:00",
+      "namespace": "namespace4",
+      "id": "72544103047",
+      "value": "value47",
+      "description": "This is metafield key47:value47 for namespace4",
+      "key": "key47",
+      "value_type": "string"
+    },
+    {
+      "created_at": "2017-04-24T20:14:24.032-04:00",
+      "updated_at": "2017-04-24T20:14:24.032-04:00",
+      "namespace": "namespace4",
+      "id": "72544103048",
+      "value": "value48",
+      "description": "This is metafield key48:value48 for namespace4",
+      "key": "key48",
+      "value_type": "string"
```

Comment:
You don't really need such a huge fixture, 2-3 metafields should be enough to test the limit parameter.